File: java-retrofit/src/main/java/com/iproov/androidapiclient/javaretrofit/ValidationResult.java
package com.iproov.androidapiclient.javaretrofit;

import android.graphics.Bitmap;
import com.google.gson.annotations.SerializedName;

public class ValidationResult {
    @SerializedName("passed")
    public Boolean isPassed;
    @SerializedName("token")
    public String token;
    @SerializedName("frame")
    public Bitmap frame;
    @SerializedName("reason")
    public String failureReason;
}

File: kotlin-fuel/src/main/java/com/iproov/androidapiclient/kotlinfuel/ValidationResult.kt
package com.iproov.androidapiclient.kotlinfuel

import android.graphics.Bitmap
import org.json.JSONObject

data class ValidationResult (
    val isPassed: Boolean,
    val token: String,
    val type: String,
    val isFrameAvailable: Boolean,
    val frame: Bitmap?,
    val failureReason: String?,
    val riskProfile: String?,
    val assuranceType: String?,
    val signals: JSONObject?
)